src/common/SwipeableFlatList/index.js

```diff
@@ -149,13 +149,14 @@ class SwipeableFlatList<TEntity> extends React.Component<
       <View>
         <FlatList
           {...rest}
-          refreshing={this.state.refreshing}
-          onRefresh={this.props.onRefresh ? this._onRefresh : null}
           extraData={this.state}
+          onEndReachedThreshold={Platform.OS === 'ios' ? 0 : 0.5}
```

Comment:
Store this as a constant and add a comment about why you need this magic number.